### ISSUE TITLE
ERROR: Referenced sniff "PHPCompatibility.FunctionUse.RemovedFunctions" does not exist

### ISSUE THREAD

**Issue by itaymesh** [extdn/github-actions-m2#110]
Hi 
We have been using extra/GitHub-actions-m2/Magento-coding-standard/8.1@master for a few months already. 
Lately, any action run ends with an error: Referenced sniff "PHPCompatibility.FunctionUse.RemovedFunctions" does not exist.
You can find the action run log [here](https://github.com/studioraz/magento2-page-builder/actions/workflows/coding-standard.yml)
I would appreciate any help.
Thank you
Itay

**Comment by sprankhub** [comment]
Weird, never seen this. Related: https://github.com/magento/magento-coding-standard/issues/390

**Comment by fredden** [comment]
I've tracked this down to a change in dependencies in the magento-coding-standard: https://github.com/magento/magento-coding-standard/commit/2a50cc7d2c671f3b1ad71b4b76faa50d4018a10f. We were setting an explicit `installed_paths` setting for PHP_CodeSniffer, but this is no longer compatible. I've switched to using the official plug-in in #111, which fixes this problem, and safe-guards us from future changes in dependencies.
